app/src/main/java/com/github/bjolidon/bootcamp/ui/authentication/AuthenticationActivity.kt

```diff
@@ -0,0 +1,100 @@
+package com.github.bjolidon.bootcamp.ui.authentication
+
+import android.content.Context
+import android.content.Intent
+import android.net.ConnectivityManager
+import android.net.NetworkCapabilities
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.firebase.ui.auth.AuthUI
+import com.firebase.ui.auth.AuthUI.IdpConfig
+import com.firebase.ui.auth.FirebaseAuthUIActivityResultContract
+import com.firebase.ui.auth.data.model.FirebaseAuthUIAuthenticationResult
+import com.github.bjolidon.bootcamp.R
+import com.google.firebase.auth.FirebaseAuth
+

```

Comment:
add documentation about the class so that we know what we have to give as intents and what the class is about